### PR TITLE
Split process creation & unverified balance mapping position

### PR DESCRIPTION
**Pull request by jordipainan**

### CODE REVIEW

File: lib/data-wrappers.ts [hunk 1]
```diff
@@ -205,7 +205,8 @@ export class ProcessContractParameters {
     maxVoteOverwrites: number;
     maxTotalCost: number;
     costExponent: number;
-    evmBlockHeight: number;
+    evmBlockHeight?: number;
+    tokenAddress?: string;
```
Comment (brickpop) [inline lib/data-wrappers.ts:209]: Duplicate of entityAddress?

File: lib/data-wrappers.ts [hunk 2]
```diff
@@ -186,15 +186,15 @@ export const processStatusValues = [
 
 // PROCESS CREATION/RETRIEVAL PARAMTERES
 
-/** Wraps and unwraps the parameters sent to `Process.newProcess()` and obtained from `Process.get()` for convenience */
+/** Wraps and unwraps the parameters sent to `Process.newProcessStd()` and to `Process.newProcessEvm()` and obtained from `Process.get()` for convenience */
 export class ProcessContractParameters {
     mode: ProcessMode;
     envelopeType: ProcessEnvelopeType;
     censusOrigin: ProcessCensusOrigin;
     entityAddress?: string;
```
Comment (brickpop) [inline lib/data-wrappers.ts:194]: Add `/** Entity or Token Address */`

**Comment by jordipainan** [comment]
@brickpop  comments addressed and added some gas usage improvements :)

**Comment by jordipainan** [comment]
closes #3

**Pull request merged**